microprofile/testing/junit5/src/main/java/io/helidon/microprofile/testing/junit5/HelidonJunitExtension.java

```diff
@@ -318,16 +315,17 @@ private void configure(ConfigMeta configMeta) {
             ConfigBuilder builder = configProviderResolver.getBuilder();
 
             configMeta.additionalSources.forEach(it -> {
-                // If not using a YAML extension, assume properties file
                 String fileName = it.trim();
-                if (YAML_SUFFIXES.stream().anyMatch(fileName::endsWith)) {
-                    builder.withSources(YamlMpConfigSource.classPath(it).toArray(new ConfigSource[0]));
-                } else {
-                    builder.withSources(MpConfigSources.classPath(it).toArray(new ConfigSource[0]));
+                String type = fileName.substring(fileName.lastIndexOf('.') + 1);
```

Comment:
As far as I can see, this is missing some original features. The original code supported both `yml` and `yaml` files, this would only support `yaml` - please make sure you replace `yml` with `yaml`

Reply:
Do we better include yml as a supported type in `YamlMetaConfigProvider` ?